samples/qa-videoplayer/src/views/MainPanel.js

```diff
@@ -49,56 +35,53 @@ const SelectableVideoPlayer = class extends Component {
 				// this.video.seek(selectionStart);
 
 				// ... or pause() and lock at end
-				this.video.pause();
+				videoRef.current.pause();
 				// this.video.seek(selectionEnd);
 			}
 		}
-	};
-
-	handleSeekOutsideSelection = (ev) => {
+	}, [selection]);
 
+	const handleSeekOutsideSelection = useCallback((ev) => {
 		// prevent the action and seek to the beginning or end
-		const {selection} = this.state;
 		const [selectionStart, selectionEnd] = selection;
 		ev.preventDefault();
 
 		if (ev.time < selectionStart) {
-			this.video.seek(selectionStart);
+			videoRef.current.seek(selectionStart);
 		} else if (ev.time > selectionStart) {
 			// this.video.pause();
-			this.video.seek(selectionEnd);
+			videoRef.current.seek(selectionEnd);
 		}
 
 		// or remove the selection and allow the default behavior
 		// this.setState({
 		// 	selection: null
 		// });
```

Comment:
can you remove this comment since it is obsolete after changing the component type?

Reply:
Done